fuse/test/file_lock_test.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2016 the Go-FUSE Authors. All rights reserved.
```

Comment:
2018

did you add your name to the AUTHORS file?

Reply:
Not sure who he is but we added his name to the authors file on his behalf.  Hope that's ok with you Kaoet?

Reply:
I missed update to AUTHORs in your commit?

Reply:
done